frontend/static/main.js

```diff
@@ -472,6 +473,7 @@ app.controller('Stats', ['$scope', '$http', '$routeParams', '$timeout', '$q', '$
                     env_params.environment = env;
                     return $http.get(stats_endpoint, { params: env_params });
                 })).then(function(data_by_env) {
+                    console.log(data_by_env);
```

Comment:
is this necessary?

Reply:
heh, of course no. I have just removed this